tests/acceptance/features/user.feature

```diff
@@ -0,0 +1,42 @@
+Feature: user
```

Comment:
In my opinion it would be better if we call this feature something else like `createAccount` or `addNewAccount`.